packages/plugin-ext/src/main/browser/workspace-main.ts

```diff
@@ -367,7 +367,7 @@ export class TextContentResource implements Resource {
             return content;
         } else {
             const content = await this.proxy.$provideTextDocumentContent(this.uri.toString());
-            if (content) {
+            if (content !== undefined && content !== null) {
```

Comment:
Minor: It might be better to simply check whether the content is a string instead of checking what it isn't.

```suggestion
            if (typeof content === 'string') {
```